public/graph.js

```diff
@@ -26,27 +26,13 @@ class GraphController {
             d.total_sent = +d.total_sent;
             d.total_pending = +d.total_pending;
             d.total_errored = +d.total_errored;
-            d.NC_received = +d.operators["NC"]["received"];
-            d.telegram_received = +d.operators["telegram"]["received"];
-            d.golis_received = +d.operators["golis"]["received"];
-            d.hormud_received = +d.operators["hormud"]["received"];
-            d.nationlink_received = +d.operators["nationlink"]["received"];
-            d.somnet_received = +d.operators["somnet"]["received"];
-            d.somtel_received = +d.operators["somtel"]["received"];
-            d.telesom_received = +d.operators["telesom"]["received"];
-            d.golis_sent = +d.operators["golis"]["sent"];
-            d.hormud_sent = +d.operators["hormud"]["sent"];
-            d.nationlink_sent = +d.operators["nationlink"]["sent"];
-            d.somnet_sent = +d.operators["somnet"]["sent"];
-            d.somtel_sent = +d.operators["somtel"]["sent"];
-            d.telesom_sent = +d.operators["telesom"]["sent"];
-            d.telegram_sent = +d.operators["telegram"]["sent"];
-            d.NC_sent = +d.operators["NC"]["sent"];
             Object.keys(d.operators)
                 .sort()
                 .forEach(key => {
                     if (!(key in operators)) {
                         operators.add(key);
+                        d[`${key}_received`] = +d.operators[key]["received"];
```

Comment:
renaming key to operator will make this more readable.

Reply:
Noted